src/js/themes/base.js

```diff
@@ -1113,6 +1113,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
             size: 'medium',
             pad: 'small',
           },
+          text: {
+            color: 'text-weak',
```

Comment:
This doesn't seem backwards compatible

Reply:
I thought we'd just inline this for DataTableColumns use cases but leave base.js as is?

Reply:
can do